bref

```diff
@@ -149,33 +122,9 @@ $app->command('cli function [--region=] [arguments]*', function (string $functio
     return (int) ($payload['exitCode'] ?? 1);
 });
 
-$app->command('info [--stage=]', function (?string $stage, SymfonyStyle $io) {
-    $stageOption = $stage ? ' --stage ' . escapeshellarg($stage) : '';
-    $io->write((new Process('serverless info'.$stageOption))->mustRun()->getOutput());
-});
-
-$app->command('remove [--stage=]', function (?string $stage, SymfonyStyle $io) {
```

Comment:
Not covered in SAM but since this is a rare use case we can document it (very simple thanks to cloudformation, either in CLI or through the AWS console).

Reply:
Documented in https://bref.sh/docs/deploy.html